### DISCUSSION TITLE
Issue downloading the openWebText2 dataset

### DISCUSSION THREAD

**Discussion by Roopa27** [fwillett/speechBCI#23] [Q&A]
I was trying to implement this repository but having issues in downloading the OpenWebText2 dataset. Can anyone help me regarding this?

**Comment by hojsad** [comment]
I have the same problem.

**Comment by yshaeri** [comment]
Try this:
wget https://paddlenlp.bj.bcebos.com/models/transformers/gpt/openwebtext2.jsonl.zst.tar

**Reply by hojsad** [reply]
Perfect. Thanks a lot.

**Reply by HanCatNguyen** [reply]
wget https://paddlenlp.bj.bcebos.com/models/transformers/gpt/openwebtext2.jsonl.zst.tar doesn't work for me either. Do you know any other way to download this OpenWebText2 dataset?

**Comment by Roopa27** [comment]
Thank you so much

________________________________
